DoctorWho/Program.cs

```diff
@@ -1,13 +1,42 @@
 ﻿using DoctorWho.Db;
+using DoctorWho.Domain;
 using Microsoft.EntityFrameworkCore;
 
-using (DoctorWhoDbContext context = new DoctorWhoDbContext())
-{
-    context.Database.EnsureCreated();
-}
+DoctorWhoDbContext.context.Database.EnsureCreated();
+
+
+//----- Execute the functions, sprocs and views ------
+
+//FunctionsViewsAndStoredProcedures.Ex_Companions_Function(1);
+
+//FunctionsViewsAndStoredProcedures.Ex_Enemies_Function(2);
+
+//FunctionsViewsAndStoredProcedures.Ex_ViewEpisodes();
+
+//FunctionsViewsAndStoredProcedures.Ex_spSummariseEpisodes();
+
+
+//----- Create new Author -----
+
+CreateUpdateDelete.CreateAuthor("New-Author");
+
+//----- Update New-Author with id = 6 -----
+
+var author = DoctorWhoDbContext.context.Authors.Find(6);
+
+author.AuthorName = "New-Author (Updated !)";
+
+Console.WriteLine(DoctorWhoDbContext.context.ChangeTracker.DebugView.LongView);
+
```

Comment:
we usually sent the new model to update, with new values, no need for complexity here